### DISCUSSION TITLE
Is CAF suitable for compute-intensive applications?

### DISCUSSION THREAD

**Discussion by phil-skillwon** [actor-framework/actor-framework#1840] [Q&A] [answered]
Is CAF suitable for compute-intensive applications? I want to use CAF instead of HPX for some compute-intensive tasks.

**Comment by Neverlord** [comment, marked as answer]
If you have compute-intense tasks, CAF would essentially just trigger your code from a message, wouldn't it? So most of the time, a compute-intense application would run in user-code. As long as you have "small messages and big computations", there won't be much stress on CAF anyways.
